js/client-battle-tooltips.js

```diff
@@ -343,7 +348,21 @@ var BattleTooltips = (function () {
 				text += '&nbsp;SpA /&nbsp;' + myPokemon.stats['spd'] + '&nbsp;SpD /&nbsp;';
 			}
 			text += myPokemon.stats['spe'] + '&nbsp;Spe</p>';
-			if (isActive) text += '<p class="section">Opponent sees:</p>';
+			if (isActive) {
+				pokemon.stats = myPokemon.stats;
+				pokemon.ability = ability;
+				pokemon.item = item;
+				pokemon.hp = myPokemon.hp;
+				pokemon.maxhp = myPokemon.maxhp;
+				pokemon.unburden = myPokemon.unburden;
```

Comment:
To clarify, .boosts is undefined for the myPokemon object, along with one or two other properties that I needed to access